config/settings.yml

```diff
@@ -14,8 +14,8 @@ moab:
   # this is a list of named storage roots.  the names are used to seed the endpoints table.
   storage_roots:
     # name: '/location/on/disk'  # e.g.
```

Comment:
sorry, just thought of one other idea:  what about a short explanatory comment like:
```ruby
# any named storage roots added in the root settings.yml will always be inherited by env specific configs, even
# if they aren't listed directly in the settings/*.yml file for an environment. so list storage roots directly in
# the relevant settings file for the env, including for unit vanilla env.
```